.github/workflows/macos-microshift.yml

```diff
@@ -13,6 +13,8 @@ on:
 jobs:
   build:
     runs-on: ${{ matrix.os }}
+    env:
+      SHELL: /bin/bash
```

Comment:
why do we need to use bash?
there's a 'standard' way of selecting the shell, `shell: bash` should work.